python/tvm/relay/frontend/coreml.py

```diff
@@ -350,6 +350,33 @@ def _MinLayerParams(op, inexpr, etab):
     return _min
 
 
+def _UnaryFunctionLayerParams(op, inexpr, etab):
+    op_type = op.type
+    if op_type == 0:
```

Comment:
Let us not use `0` / `1` / `2` ... directly. We could use ` op.THRESHOLD` / `op.EXP` and so on. See https://apple.github.io/coremltools/coremlspecification/sections/NeuralNetwork.html#unaryfunctionlayerparams